components/brave_geolocation_permission/common/BUILD.gn

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//mojo/public/tools/bindings/mojom.gni")
+
+mojom("brave_geolocation_permission") {
+  sources = [ "brave_geolocation_permission.mojom" ]
+
+  export_class_attribute_blink = "CORE_EXPORT"
+  export_define_blink = "BLINK_CORE_IMPLEMENTATION=1"
+  export_header_blink = "third_party/blink/renderer/core/core_export.h"
```

Comment:
please remove these export variables and add `brave_geolocation_permission_blink` directly into blink core via `brave_blink_renderer_core_deps`.

Reply:
Addressed via 17ba9845d06515106ea3221f19c3192e4da9e1bd but it seems above defines are needed. 
Got build failure w/o them.